docs/AboutUs.md

```diff
@@ -9,51 +9,51 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Tan Jin Waye
 
 <img src="images/johndoe.png" width="200px">
 
 [[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
+[[github](https://github.com/Tan-Jin-Waye)]
 [[portfolio](team/johndoe.md)]
 
-* Role: Project Advisor
+* Role: TBC
```

Comment:
Since this role is TBC, suggest adding back Responsibilities.

Reply:
good point will create new pr